libraries/stdlib/js/test/core/StringsJsTest.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package test.js
+
+import kotlin.test.*
+
+class StringsJsTest {
```

Comment:
Better to move common tests into `libraries/stdlib/test/text/StringTest.kt` instead of duplicating them for each platform.